windows/devices/uwb/include/windows/devices/uwb/IUwbSessionDdi.hxx

```diff
@@ -46,7 +46,8 @@ struct IUwbSessionDdi
     SessionGetRangingCount(uint32_t sessionId) = 0;
 
     // IOCTL_UWB_SESSION_UPDATE_CONTROLLER_MULTICAST_LIST
-    virtual std::future<::uwb::protocol::fira::UwbSessionUpdateMulicastListStatus>
+    // This function requires that the mac addresses are short

```

Comment:
This is so strange! But I did confirm that's accurate. I wonder how a session with controlees using extended addresses is updated 😕

Reply:
I think this implies that for multicast session, they must use short addresses (or they can't be dynamically updated)

Reply:
Good call, that makes sense.